src/plugins/analysis/binwalk/requirements.txt

```diff
@@ -1,3 +1,2 @@
 capstone==4.0.2
 cstruct==4.0
-matplotlib==3.6.2
```

Comment:
I don't like removing the dependency from the plugin dependency file.

If we removed the matplotlib dependency from FACT the plugin would stop working.

Reply:
Yes you are probably right
restored the matplotlib dependency in binwalk and the frontend requirements file